src/main/java/org/fundacionjala/at13/students/SaulCaspa.java

```diff
@@ -9,53 +9,102 @@
+     */
     private String programmingLanguages;
-    private String residenceCity;
-
-    public SaulCaspa(){
-        this.name="Saul Caspa Miranda";
-        this.nickName="Saul";
-        this.age=26;
-        this.career="Systems Engineering";
-        this.university="Universidad Mayor de San Simon";
-        this.programmingLanguages="Java, php, python";
-        this.residenceCity="Cochabamba";
-    }
-    public String getResidenceCity(){
-        return "Currently I live in "+residenceCity+".";
-    }
-    public String ShowNickName(){
-        return "I like to be called "+this.nickName;
-    }
-    
-    public String showStudentsInformation() {
-        return "Hello "+"my name is " + name + "\n" +
-                "I like to be called '" + nickName + "\n" +
-                "I am "+ age +" years old" + "\n" +
-                "I am " + career + "\n" +
-                "I study at " + university+ " university \n" +
-                "My programming languages are " + programmingLanguages +"\n" +
-                "My residence city " + residenceCity +"\n" +
-                "I have born in the year "+(2021-age)+".";
-    }
-    public String ShowName(){
-        return "Hi, my name is "+this.name;
-    }
-    public String showProgramingLanguage() {
+    /**
+     * The "residenceCity" of this student.
+     */
+    private String residenceCity = "Cochabamba";
+    /**
+     * The "currentYear".
+     */
+    private final int currentYear = 2021;
+    /**
+     * Class SaulCaspa contains the information for this student.
+     */
+    public SaulCaspa() {
+        this.name = "Saul Caspa Miranda";
+        this.nickName = "Saul";
+        this.career = "Systems Engineering";
+        this.residenceCity = "Cochabamba";
+        this.university = "Universidad Mayor de San Simon";
+        this.programmingLanguages = "Java, php, python";
+    }
+    /**
+     * @return the residence city.
+     */
+    public final String getResidenceCity() {
+        return "Currently I live in " + this.residenceCity + ".";
+    }
+    /**
+     * @return the nickname.
+     */
+    public final String showNickName() {
+        return "I like to be called " + this.nickName;
+    }
+    /**
+     * @return all student information.
+     */
+    public final String showStudentsInformation() {
+        return "Hello " + "my name is " + name + "\n"
+        + "I like to be called " + nickName + "\n"
+        + "I am " + age + " years old" + "\n"
+        + "I am " + career + "\n"
+        + "I study at " + university + " university \n"
+        + "My programming languages are " + programmingLanguages + "\n"
+        + "My residence city " + residenceCity + "\n"
+        + "I have born in the year " + (currentYear - this.age) + ".";
+    }
+    /**
+     * @return name.
+     */
+    public final String showName() {
+        return "Hi, my name is " + this.name;
+    }
+    /**
+     * @return favourite programming language.
```

Comment:
Maybe change of favourite to favorite

Reply:
Favourite is as valid as favorite since they are from british and american english.